core/dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -51,6 +51,29 @@
 
 {% endmacro %}
 
+{% macro get_create_index_sql(relation, index_dict) -%}
+  {{ return(adapter.dispatch('get_create_index_sql')(relation, index_dict)) }}
+{% endmacro %}
+
+{% macro default__get_create_index_sql(relation, index_dict) -%}
+  {{ exceptions.raise_not_implemented(
+    'get_create_index_sql macro not implemented for adapter '+adapter.type()) }}
+{% endmacro %}
+
+{% macro create_indexes(relation) -%}
+  {{ adapter.dispatch('create_indexes')(relation) }}
+{%- endmacro %}
+
+{% macro default__create_indexes(relation) -%}
+  {%- set _indexes = config.get('indexes', default=[]) -%}
```

Comment:
Should I just call `config.get('indexes')` as I do now or create a method on the config like `config.persist_relation_docs()`?

Reply:
I think `persist_relation_docs` just exists to validate that `persist_docs` is a dict. We can do that in other ways, e.g. via methods to the Postgres class object in python (see other comments).